### PR TITLE
Update sanctuary-scripts to version 1.5.0

### PR DESCRIPTION
**Pull request by Avaq**

### CODE REVIEW

**Comment by Avaq** [comment]
@davidchambers could you lend a hand? 

This code example:

https://github.com/Avaq/permissionary/blob/5cd9614f959145268d2b1945a9ae311d929e87a0/index.js#L67-L94

...gets transformed to this for validation:

```js
// This defines a mapping from roles to permissions.
// We can use wildcards to assign multiple permissions at once.
var createVerifier = checkPermission({
  'content-reader': ['content.read', 'images.read'],
  'content-writer': ['content.write', 'images.upload'],
  'superadmin': ['*']
});
;
// Let's say our user Bob is a content-reader, and also a content-writer.
var canBob = createVerifier(['content-reader', 'content-writer']);
;
// And Alice is an administrator.
var canAlice = createVerifier(['superadmin']);
;
// Bob has this permission through his content-reader role.
canBob('content.read');
true;

// Bob does not have this permission.
canBob('users.create');
false;

// Alice, however, does. She has all permissions (even the ones
// we haven't thought of yet).
canAlice('users.create')
true
```

...which causes linting errors for the `no-extra-semi`, `semi-style`, and `semi` rules.

Least clear to me is why the last statement doesn't have a semicolon.

**Comment by davidchambers** [comment]
The interim fix is to disable these rules for `*.md` files as I did in fluture-js/fluture-sanctuary-types#26. The long-term fix is to update [`rewrite`][1] to insert semicolons more discerningly.


[1]: https://github.com/sanctuary-js/sanctuary-scripts/blob/v1.5.0/bin/lint-readme#L16-L41

**Comment by Avaq** [comment]
> The interim fix is to disable these rules for `*.md` files

I'm okay with that for now. I wanted to be sure I'm using transcribe correctly. I created https://github.com/sanctuary-js/sanctuary-scripts/issues/14 as a reminder.

File: README.md [hunk 1]
```diff
@@ -40,7 +41,7 @@ assign both roles to that user.
 
 ## API
 
-<h4 name="checkPermission"><code><a href="https://github.com/Avaq/permissionary/blob/v0.1.0/index.js#L49">checkPermission :: StrMap (Array String) -⁠> Array String -⁠> String -⁠> Boolean</a></code></h4>
+#### <a name="checkPermission" href="https://github.com/Avaq/permissionary/blob/v0.0.0/index.js#L50">`checkPermission :: StrMap (Array String) -⁠> Array String -⁠> String -⁠> Boolean`</a>
```
Comment (davidchambers) [inline README.md:44]: What happened to the version number? :scream:

Comment (Avaq) [reply README.md:44]: Oh darn it.. I committed the README I was generating locally to identify the issue. I should be more careful and less hurried. I will push a fix to `master` as if running `sanctuary-release`. >.<

Comment (Avaq) [reply README.md:44]: Actually I'll just release a patch for consistency.

**Comment by davidchambers** [comment]
> Least clear to me is why the last statement doesn't have a semicolon.

The leading `> ` is missing. ;)

**Comment by davidchambers** [comment]
I have a fix ready for the missing `> `, but we should resolve #10 first. ;)

**Comment by davidchambers** [comment]
I just opened #11 to add the missing `> `.

**Pull request merged**